### ISSUE TITLE
bug(mat-select): Multiselect list | active item has highlight issue

### ISSUE THREAD

**Issue by SergiyCh** [angular/components#26740]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

In multiselect list if an item is checked - it is not highlighted during arrow UP / DOWN navigation, i.e. user can't see what item is currently active.

### Reproduction

Steps to reproduce:
1. Create a multiselect list (with checkboxes)
2. Navigate through the list using arrow keys.
3. Check some items using space.

NOTE: issue is in Chrome, in EDGE looks like all OK.

### Expected Behavior

Each item is highlighted duting navigation

### Actual Behavior

Actual results: checked items are not highlighted.

### Environment

- Angular:
- CDK/Material:
- Browser(s):
- Operating System (e.g. Windows, macOS, Ubuntu):

**Comment by zarend** [comment]
Hello @SergiyCh ,

Thank you for reporting. I am unable to reproduce on the examples at https://material.angular.io/components/select/overview. We need more information to determine if this issue requires action.

Could you please provide a screenshot, or even better, a stackblitz reproduction?

Best,

Zach

**Comment by SergiyCh** [comment]
[screen-capture (1).webm](https://user-images.githubusercontent.com/87659772/229752776-b5ba2360-ee9e-402c-bfc3-9adc20f2584b.webm)

Here is the video of the issue. As you can see - selected items are not highlighted during navigation

**Comment by SergiyCh** [comment]
@zarend

**Comment by zarend** [comment]
Hello,

Thank you for getting back to me. I can reproduce on https://material.angular.io/components/select/examples.


Although, I cannot reproduce on https://next.material.angular.io/components/select/examples, so I believe this issue has already been fixed. Looks like this fix is available in the `next` release, and I would expect a fix to go out in the upcoming major release.

-Zach
